composer.json

```diff
@@ -101,10 +100,8 @@
         "zendframework/zendsearch": "@dev"
     },
     "conflict": {
-        "symfony/symfony": "2.8.10",
-        "symfony/phpunit-bridge": "2.8.10",
-        "symfony/monolog-bundle": "2.8.10",
-        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1",
+       "doctrine/lexer": "1.0.0",
```

Comment:
needed to add this because I run into a strange error https://github.com/composer/composer/issues/7161

/cc @danrot